### ISSUE TITLE
[a11y]: DefinitionTooltip content not announced by JAWS

### ISSUE THREAD

**Issue by wkeese** [carbon-design-system/carbon#19285]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.80

### React version

18

### Automated testing tool and ruleset

manual test

### Assistive technology

JAWS

### Description

Apparently, JAWS doesn’t announce the definitions for Carbon’s DefinitionTooltip.

But, I noticed that the content is missing `aria-live="polite"`, so maybe that's the only problem.
Contrast with [Toggletip](https://react.carbondesignsystem.com/?path=/story/components-toggletip--default) which does have `aria-live="polite"`.

My other hypothesis was that it was because JAWS doesn’t expect a mere focus change to trigger a content change.  Hopefully that's not the issue as it would be hard to fix without removing the current open-on-focus behavior.

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-definitiontooltip--default

### Steps to reproduce

Just use JAWS on the DefinitionTooltip story (https://react.carbondesignsystem.com/iframe.html?globals=&args=&id=components-definitiontooltip--default&viewMode=story).

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
